server/services/reportService.ts

```diff
@@ -152,6 +152,54 @@ export default class ReportService {
     }
   }
 
```

Comment:
This updates both the reasonsForUseOfForce and useOfForceDetails sections in the DB plus inserts a new row to report_edit